projects/BEVFusion/bevfusion/transfusion_head.py

```diff
@@ -725,10 +724,11 @@ def get_targets_single(self, gt_instances_3d, preds_dict, batch_idx):
                 center_int = center.to(torch.int32)
 
                 # original
-                # draw_heatmap_gaussian(heatmap[gt_labels_3d[idx]], center_int, radius) # noqa: E501
+                draw_heatmap_gaussian(heatmap[gt_labels_3d[idx]], center_int,
+                                      radius)  # noqa: E501
                 # NOTE: fix
-                draw_heatmap_gaussian(heatmap[gt_labels_3d[idx]],
-                                      center_int[[1, 0]], radius)
+                # draw_heatmap_gaussian(heatmap[gt_labels_3d[idx]],
```

Comment:
Please remove lines 729-731 since it's unnecessary.

Reply:
remove these lines in new transfusion head at https://github.com/open-mmlab/mmdetection3d/pull/2547/commits/bb0a95e7be5c4eaaf318d6af3f07f40c370c1cca.
Also, I renamed head in bevfusion due to name conflict at https://github.com/open-mmlab/mmdetection3d/pull/2547/commits/dc948cc88b7730974eb1f0550cd37e2f36bf244b. Please let me know if this change is not applicable.